tidy3d/components/base_sim/simulation.py

```diff
@@ -102,6 +108,14 @@ class AbstractSimulation(Box, ABC):
         description="String specifying the front end version number.",
     )
 
+    plot_units: Optional[LengthUnit] = pd.Field(
+        None,
```

Comment:
should it be default of `"um"`?

Reply:
I could do that, but it will change the plot labels from previous versions. It would now include the "(um)" in axis labels, which was not added in previous tidy3d. When it is set to "None", the behavior should be exactly the same as previous tidy3d versions.

Let me know what you prefer

Reply:
hm, I feel like probably they should always say "um"? so maybe I'd be in favor of simply including them from 2.8 onwards. we'll see what others say.

Reply:
to clarify, always say "um" by default..

Reply:
I think it is a good idea to always have the unit label to avoid confusion especially if we are going to support other units. Perhaps instead of "um", it can be the more technically correct "μm".